chore: Document how to use form controls in a Field (Set) (#1548)
Amsterdam/design-system: storybook/src/components/FieldSet/FieldSet.docs.mdx

### Radio group

Use a Field Set to group several Radios with a legend, description, and / or an error message.

> When grouping Radios, add `role="radiogroup"` to the Field Set to have it explicitly announced as a radio group (the default role is just ‘group’).

Reviewer: No need to repeat the grouping once more.

```suggestion
Add `role="radiogroup"` to the Field Set to have it explicitly announced as a radio group (the default role is just ‘group’).
```

The ‘radio group’ role also allows using `aria-required` on Field Set; that isn’t allowed for the ‘group’ role.

> Always also set `aria-required` on the individual Radios though, to make sure it’s read by screen readers.

Reviewer: ‘Always also’ and ‘though’ feel a bit off. Grammarly suggests:

```suggestion
However, always set `aria-required` on the individual Radios to ensure that screen readers can read it.
```

We could replace ‘However’ with ‘Still’ or ‘Even so’.

Author: I like 'Even so', it makes it clearer that you also have to set `aria-required` on the individual Radios

<Canvas of={FieldSetStories.RadioGroup} />

> ### Radio group with error

Reviewer: Should we also say ‘With Validation’ here, as we do for the Form components?
